### DISCUSSION TITLE
[Question] Difference between tactile sensor and support contact sensor

### DISCUSSION THREAD

**Discussion by ruoshiwen** [isaac-sim/IsaacLab#1143] [Q&A]
### Tactile sensors' support
I read your paper on Isaac Orbit. In the future work section, you mentioned "Further enhancements to the framework include integrating tactile sensors and 6-axis force-torque sensors."
I saw that Orbit supports contact sensors and my understanding is they measure the contact forces between rigid bodies. My questions is what is the difference between the already supported contact force sensors and the said tactile sensors? Are these tactile sensors specifically referring to Gelsight family sensors?

Look forward to your answer.

**Comment by pascal-roth** [comment]
@Mayankm96 can you elaborate on that?

**Comment by Mayankm96** [comment]
Hi @ruoshiwen ,

Yes, this corresponds to actual tactile sensors (such as Gelsight). Contact sensors only give you the net external normal contact force on a body. This isn't the same as what a tactile sensor or force-torque/haptic sensor gives.
